x-pack/plugins/actions/server/lib/ensure_sufficient_license.ts

```diff
@@ -6,9 +6,10 @@
 import { ActionType } from '../types';
 import { LICENSE_TYPE } from '../../../licensing/common/types';
 import { ServerLogActionTypeId, IndexActionTypeId } from '../builtin_action_types';
-import { CASE_ACTION_TYPE_ID } from '../../../case/server';
 import { ActionTypeConfig, ActionTypeSecrets, ActionTypeParams } from '../types';
 
+// Formerly exported from `case` plugin
+const CASE_ACTION_TYPE_ID = '.case';
```

Comment:
I think as a quick fix it's also fine 👍 
In `x-pack/plugins/case/server/connectors/case/index.ts` I'd probably also import `CASE_ACTION_TYPE_ID` from `actions` plugin instead of duplicating it. So it would basically invert the dependency.

From the software design it looks trickier. Again, I know nothing about how `actions` plugin is designed, but from what I see it knows about concrete types of actions. It just happened that `CASE_ACTION_TYPE_ID` was declared in `case` plugin. If this is the intended design, I think it's ok. The downsides:
- "actions" code is conceptually separated between `actions` plugin and clients like `case`
- it's not super scalable if number of actions will grow significantly

If there's a goal to make it scalable, I'd maybe consider thinking about some "actions registration" API in the `actions` plugin, and defining userland actions (on top of core actions implemented inside `actions`) in plugins like `case`.

Hopefully guys who worked on `actions` could give a better feedback.

Reply:
@banderror 

Case conenctor is registered to `actions` through "actions registration" API 😄 . `actions` expose an API for other plugins to be able to register their connector at will.

The downside with this fix is that if in the feature Cases change the id of the connector the license check will fail. But it is very unlikely that this will happen.

Reply:
@cnasikas @banderror Thanks for the feedback.
I'm curious if any integrations tests would catch a change to `CASE_ACTION_TYPE_ID`. I'll check and report back.

Reply:
@oatkiller I just noticed that this PR https://github.com/elastic/kibana/pull/88556/files#diff-e1393c378d9aa36c8371baa1cfc42541bf82a68a9ba377594a96daebd591b554R11 made the same change as you did. The alerting team seems to be fine with it 😄.

Reply:
Thanks for pointing that out. It seems that PR copy and exported the `CASE_ACTION_TYPE_ID` string, but that export doesn't seem to be used anywhere. I supposed we could look import those constants from the alerting plugin.